ui/src/styles/_carbon.scss

```diff
@@ -57,12 +57,20 @@
   flex-direction: column;
 }
 
-.aside__container--text {
-  padding: $spacing-05;
-
-  @media (min-width: 1056px) {
-    padding-left: 15%;
-    padding-right: $layout-05;
+.aside__container {
+  &--text {
+    padding: $spacing-05;
+
+    @media (min-width: 1056px) {
+      padding-left: 15%;
+      padding-right: $layout-05;
+    }
+    .bx--link {
+      color: $carbon--gray-10;
```

Comment:
This used to be blue but accessibility checker flagged the lack of contrast between blue on black background.